halle

```diff
@@ -4,13 +4,20 @@ asn: 65046
 networks:
   ipv4:
     - 10.62.0.0/16
+  ipv6:
+    - fd0f:8db3:d50a::/64
 bgp:
 #  halle1:
 #    ipv4: 10.207.0.13
-  halle2:
-    ipv4: 10.207.0.14
+#  halle2:
+#    ipv4: 10.207.0.14
```

Comment:
This removes all bgp peers and violates the yaml syntax.

If you're not peering on ICVPN remove the bgp block altogether.